src/cpp/session/modules/SessionEnvironment.R

```diff
@@ -432,7 +432,7 @@
    obj <- get(objName, env)
    # objects containing null external pointers can crash when
    # evaluated--display generically (see case 4092)
-   hasNullPtr <- .rs.hasNullExternalPointer(obj)
+   hasNullPtr <- .Call("rs_hasExternalPointer", obj, TRUE)
```

Comment:
You might want to specify `PACKAGE = "(embedding)"` as otherwise R will search through all loaded DLLs for this routine and this is often slow. Since we're calling this on a potentially large environment it's worth being explicit there.